apps/fluent-tester/src/FluentTester/TestComponents/ContextualMenu/ContextualMenuTest.tsx

```diff
@@ -309,12 +277,7 @@ const IconContextualMenu: React.FunctionComponent<{}> = () => {
             <Text>Menu Visibility: </Text>
             {isContextualMenuVisible ? <Text style={{ color: 'green' }}>Visible</Text> : <Text style={{ color: 'red' }}>Not Visible</Text>}
           </Text>
-
-          <Text>
-            <Text>Submenu Visibility: </Text>
-            {isSubmenuVisible ? <Text style={{ color: 'green' }}>Visible</Text> : <Text style={{ color: 'red' }}>Not Visible</Text>}
-          </Text>
-          <Button style={{ borderColor: 'transparent' }} icon={{ svgSource: svgProps, width: 12, height: 12 }} onClick={toggleShowContextualMenu} componentRef={stdBtnRef} />
+          <Button style={{ borderWidth: 0, width: 32}} icon={{ svgSource: svgProps, width: 12, height: 12 }} onClick={toggleShowContextualMenu} componentRef={stdBtnRef} />
```

Comment:
If minWidth is set to 32, is setting width to 32 here necessary?

Reply:
good point!